app/androidutils.cpp

```diff
@@ -214,6 +226,8 @@ void AndroidUtils::handleActivityResult( int receiverRequestCode, int resultCode
     QString absolutePath = absolutePathJNI.toString();
 
     QString selectedImagePath = "file://" + absolutePath;
+    getExifInfo( absolutePath );
```

Comment:
Do we want to use it all the time? Not only if user sets it in settings or something like that?